### PR TITLE
Show 100000+ in stats when there are more than 100000 alert groups in the result

### PR DESCRIPTION
**Pull request by iskhakov**
# What this PR does

## Checklist

- [ ] Unit, integration, and e2e (if applicable) tests updated
- [ ] Documentation added (or `pr:no public docs` PR label added if not required)
- [ ] `CHANGELOG.md` updated (or `pr:no changelog` PR label added if not required)

### CODE REVIEW

File: engine/apps/api/views/alert_group.py [hunk 1]
```diff
@@ -398,20 +398,14 @@ def enrich(self, alert_groups):
 
     @action(detail=False)
     def stats(self, *args, **kwargs):
-        alert_groups = self.filter_queryset(self.get_queryset())
-        # Only count field is used, other fields left just in case for the backward compatibility
+        MAX_COUNT = 1001
```
Comment (iskhakov) [inline engine/apps/api/views/alert_group.py:401]: TODO: define this number

File: engine/apps/api/views/alert_group.py [hunk 2]
```diff
@@ -398,20 +398,14 @@ def enrich(self, alert_groups):
...
+        alert_groups = self.filter_queryset(self.get_queryset())[:MAX_COUNT]
+        count = alert_groups.count()
```
Comment (iskhakov) [inline engine/apps/api/views/alert_group.py:403]: check if we need to use len() here

File: engine/apps/api/views/alert_group.py [hunk 3]
```diff
@@ -398,20 +398,14 @@ def enrich(self, alert_groups):
...
-                "count": alert_groups.filter().count(),
-                "count_previous_same_period": 0,
-                "alert_group_rate_to_previous_same_period": 1,
-                "count_escalations": 0,
-                "count_escalations_previous_same_period": 0,
-                "escalation_rate_to_previous_same_period": 1,
-                "average_response_time": None,
-                "average_response_time_to_previous_same_period": None,
-                "average_response_time_rate_to_previous_same_period": 0,
-                "prev_period_in_days": 1,
```
Comment (joeyorlando) [inline engine/apps/api/views/alert_group.py:414]: 👍

File: engine/apps/api/views/alert_group.py [hunk 4]
```diff
@@ -398,20 +398,14 @@ def enrich(self, alert_groups):
 
     @action(detail=False)
     def stats(self, *args, **kwargs):
-        alert_groups = self.filter_queryset(self.get_queryset())
-        # Only count field is used, other fields left just in case for the backward compatibility
+        MAX_COUNT = 1001
+        alert_groups = self.filter_queryset(self.get_queryset())[:MAX_COUNT]
+        count = alert_groups.count()
+        print(count)
```
Comment (joeyorlando) [inline engine/apps/api/views/alert_group.py:404]: `print` statement

**Review by joeyorlando — APPROVED** [review]

File: engine/apps/api/views/alert_group.py (hunk 3 above)
Comment (Matvey-Kuk) [reply engine/apps/api/views/alert_group.py:416]: Niiiice

**Pull request merged**